src/library_pthread.js

```diff
@@ -239,7 +239,15 @@ var LibraryPThread = {
       pthread.stackBase = 0;
       if (pthread.worker) pthread.worker.pthread = null;
     },
-
+    returnThreadToPool: function(workerToFree) {
```

Comment:
Let's rename this to `returnWorkerToPool` (threads are never returned to the pool, threads are quit, and then the worker that was hosting the thread will return to the pool) - also `workerToFree` -> `worker`

Reply:
Yeah good point.